### DISCUSSION TITLE
TrailingStopLoss

### DISCUSSION THREAD

**Discussion by grumpydolphin** [cyberjunky/3commas-cyber-bots#156] [Q&A]
Do I understand this correctly.

For example, you could configure the script to trigger at 1% TP, and set the trailing SL at 0%?

Than it can trail the TP and SL upwards?

**Comment by amargedon** [comment]
Yes, that is correct. But, be aware that a SL on 0% could leave you with a loss for the deal! This due to exchange fees and a SL order is a market order instead of a limit order. That's why I always set the SL on 0.30% or higher initially.
